server-adapters/resteasy-reactor-netty/src/test/java/org/jboss/resteasy/plugins/server/reactor/netty/UriInfoTest.java

```diff
@@ -95,7 +95,7 @@ public void testCannotDetermineHost() {
         try {
             extractor.extract(req, "/contextPath");
             fail("Inability to determine a host address should have thrown an IllegalArgumentException.");
-        } catch (final AssertionError ae) {
+        } catch (final IllegalArgumentException ignore) {
```

Comment:
Ugh, this was an accidental push:(

Reply:
No worries, I got the test working at least. 5.0.5.Final has been released too.